src/ut/authentication_test.cpp

```
@@ -2828,21 +2890,24 @@ TEST_F(AuthenticationTest, AuthSuccessTimerDelete)
   // The authentication module lets the request through.
   auth_sproutlet_allows_request();
 
+  // Verify the chronos_connection expectations here, as otherwise they wait until the test class is deleted
+  testing::Mock::VerifyAndClear(MockChronosConnectionHelper::get_chronos_connection());
   // clean up
   _hss_connection->delete_result("/impi/6505550001%40homedomain/av?impu=sip%3A6505550001%40homedomain&server-name=sip%3Ascscf.sprout.homedomain%3A5058%3Btransport%3DTCP");
 }
 
 // Assert that we don't send a delete if the timer creation failed.
-TEST_F(AuthenticationTest, AuthSuccessTimerCreationFail)
+TEST_F(AuthenticationTimerTest, AuthSuccessTimerCreationFail)
 {
+
   pjsip_tx_data* tdata;
   // Set up the HSS response for the AV query using a default private user identity.
   _hss_connection->set_result("/impi/6505550001%40homedomain/av?impu=sip%3A6505550001%40homedomain&server-name=sip%3Ascscf.sprout.homedomain%3A5058%3Btransport%3DTCP",
                               "{\"digest\":{\"realm\":\"homedomain\",\"qop\":\"auth\",\"ha1\":\"12345678123456781234567812345678\"}}");
 
   // Set up the timer creation to return an error, simulating timer creation failure.
-  EXPECT_CALL(*_chronos_connection, send_post(_,_,"/authentication-timeout",_,_,_))
-              .WillOnce(DoAll(SetArgReferee<0>("TIMER"),
+  EXPECT_CALL(*MockChronosConnectionHelper::get_chronos_connection(), send_post(_,_,"/authentication-timeout",_,_,_))
+              .WillOnce(DoAll(SetArgReferee<0>(""),
```

Minor things, but we technically shouldn't be doing `SetArgReferee<0>("")` here, we should just return the error without setting the first argument